tests/cypress/support/templates/cats.group/index.html

```diff
@@ -1,4 +1,4 @@
-{exp:channel:categories channel="news" style="linear"}
```

Comment:
Linear and nested categories use different code that generates display. This test template is specifically set to use linear mode, so please let's continue using it. And maybe we need separate test template for the nested mode.